common/components/chrome/MainHeader.jsx

```diff
@@ -53,437 +41,160 @@ class MainHeader extends React.Component<{||}, State > {
-              style={{ transformOrigin: 'top' }}
-            >
-              <Paper className="SubHeader-dropdown-menu">
-                <ClickAwayListener onClickAway={this._closeDropdown}>
-                  <MenuList>
-                    <p className="SubHeader-dropdown-name">{`${CurrentUser.firstName()} ${CurrentUser.lastName()}`}</p>
-                    <MenuItem onClick={(e) => this.navigateToSection(e, 'EditProfile')}>My Profile</MenuItem>
-                    {this.state.showMyProjects && <MenuItem onClick={(e) => this.navigateToSection(e, 'MyProjects')}>My Projects</MenuItem>}
-                    {this.state.showMyGroups && <MenuItem onClick={(e) => this.navigateToSection(e, 'MyGroups')}>My Groups</MenuItem>}
-                  </MenuList>
-                  <Divider />
-                  <a href="/logout" onClick={this._closeDropdown}>
-                    <MenuItem>
-                      Sign Out
-                    </MenuItem>
-                  </a>
-                </ClickAwayListener>
-              </Paper>
-            </Grow>
-          )}
-        </Popper>
-      </div>
+      <Navbar collapseOnSelect expand="lg" bg="navlight" variant="light">
+        <Navbar.Brand><a href={url.section(Section.Home)}><img src={cdn.image("dl_logo.png")} alt="DemocracyLab" /></a></Navbar.Brand>
+        {CurrentUser.isLoggedIn() ? null : <Button className="MainHeader-showmobile MainHeader-login-button" variant="outline-primary" href={url.section(Section.LogIn, url.getPreviousPageArg())}>Log In</Button>}
+        <Navbar.Toggle aria-controls="nav-pagenav-container" />
+        <Navbar.Collapse id="nav-pagenav-container" className="flex-column">
+          <Nav className="MainHeader-usernav ml-auto">
+            {CurrentUser.isLoggedIn() ? this._renderUserSection() : this._renderLogInSection()}
+          </Nav>
+          <Nav className="MainHeader-pagenav mr-auto">
+            {this._renderNavLink(Section.Home, "Home")}
+            <NavDropdown title="Projects" id="nav-projects">
+              {this._renderNavDropdownItem(Section.FindProjects, "Find Projects")}
+              {this._renderNavDropdownItem(Section.CreateProject, "Create Project")}
+            </NavDropdown>
+            <NavDropdown title="Events" id="nav-events">
+              {this._renderNavDropdownItem(Section.FindEvents, "Find Events")}
+              {this._renderNavDropdownItem(Section.CreateEvent, "Create Event")}
+            </NavDropdown>
+            <NavDropdown title="About" id="nav-about">
+              {this._renderNavDropdownItem(Section.AboutUs, "About Us")}
+              {this._renderNavDropdownItem(Section.ContactUs, "Contact Us")}
+              {this._renderNavDropdownItem(Section.Press, "News")}
+            </NavDropdown>
+            {window.BLOG_URL ? <Nav.Link href={window.BLOG_URL}>Blog</Nav.Link> : null}
+            {window.EVENT_URL ? <Nav.Link href={_.unescape(window.EVENT_URL)}>Live Event</Nav.Link> : null}
```

Comment:
```suggestion
            {window.EVENT_URL ? <Nav.Link href={_.unescape(window.EVENT_URL)}>Event</Nav.Link> : null}
```

Should just be 'Event' since we show this even when the event is not live.